### DISCUSSION TITLE
Changes after ImageMagick 7.1.1-27 ?

### DISCUSSION THREAD

**Discussion by herb52** [ImageMagick/ImageMagick#7412] [Help]
### ImageMagick version

7.1.1-33

### Operating system, version and so on

Windows 10 - 64bit

### Description

In order to generate thumbnails/previews I use convert.exe from e.g ImageMagick-7.1.1-27-portable-Q16-HDRI-x64.zip
At the moment I work with *.avif and also *.jxl images.

I use the following "generic" command:
`convert.exe -quiet -strip -identify -resize 1920x1920> -clamp -depth 8 test.avif test.tiff `

Version 7.1.1-27 works without any problems, but
with version 7.1.1-33 I get the following error:
`convert_7.1.1.33.exe: NoImagesFound `-strip' at CLI arg 2 @ error/operation.c/CLIOption/5479.`

This problem does not depend on any specific file.

Are there changes after version 7.1.1-27 which I have overseen or is it a general problem?
Thanks for your help in advance
Best regards
herb

**Comment by dlemstra** [comment]
Thanks for reporting this. I think we will need to update the portable edition to no longer distribute the `convert.exe` file. It now uses the `magick.exe` logic that requires you to reorder your options. You will need to do this instead: `magick.exe -quiet test.avif -strip -identify -resize 1920x1920> -clamp -depth 8 test.tiff`.

And I suspect we will also need to check our other releases to make sure we show a warning message when someone uses `convert.exe`.

**Reply by herb52** [reply]
Hello,

thanks for your quick reply.
I followed your advice and it works as expected.

BUT: Why is also a file with name "-clamp" created with following content
`test.avif AVIF 1280x720 1280x720+0+0 10-bit sRGB 250068B 0.147u 0:00.147`

Best regards
herb

**Reply by dlemstra** [reply]
I suspect this is being redirected `-resize 1920x1920>`. Please respond here what you had to do to get this fixed. I think you can fix this with `"1920x1920>"` (add `"`).

**Reply by herb52** [reply]
Hello

thanks again for your quick reply.

> I suspect this is being redirected -resize 1920x1920>
>
Yes you are a genius.
I copied the command as it is used inside my application. ">" is part of syntax for -resize.
 Therefore I totally forgot that ">" has a special meaning with cmd.exe

Thanks again and
best regards
herb
